app/src/main/java/io/branch/branchster/BranchsterAndroidApplication.java

```diff
@@ -13,7 +14,7 @@
 public class BranchsterAndroidApplication extends MultiDexApplication {
     public void onCreate() {
         super.onCreate();
-        Branch.enableLogging();
+        Branch.enableLogging(BranchLogger.BranchLogLevel.VERBOSE);
```

Comment:
Is this necessary? I thought the default would be verbose if not set

Reply:
The default is DEBUG

Reply:
Alright, a little quirky imho but we can proceed with this.